src/pages/profile-edit.page.js

```diff
@@ -47,17 +57,30 @@ export default class ProfileEditPage extends Component {
   }
 
   async componentDidMount() {
-    this.setState({loading: true});
+    this.setState({ loading: true });
     const currState = this.context.appState;
-
-    this.setProfile(currState);
+    this.fetchLocationPermissions();
     this.fetchHealthWorkerStatuses();
     this.fetchSymptoms();
+    this.setProfile(currState);
+  }
+
+  async fetchLocationPermissions() {
+    // Checks if user allowed geolocation on page load; disables location toggle if blocked
+    // NOTE* This feature is only supported by Chrome and Edge atm
+    // (https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API)
+    if (navigator && navigator.permissions) {
+      navigator.permissions.query({ name: 'geolocation' }).then((PermissionStatus) => {
```

Comment:
lowercase

Reply:
i think in general we need a better way of messaging the user when location is blocked by browser already. its really hard to reset, too

Reply:
![image](https://user-images.githubusercontent.com/3998637/80287998-e59a4b00-8702-11ea-8527-07cb49ee76cf.png)